StripeConnect/StripeConnect/Source/Internal/AuthenticatedWebView/AuthenticatedWebViewManager.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  AuthenticatedWebViewManager.swift
+//  StripeConnect
+//
+//  Created by Mel Ludowise on 10/15/24.
+//
+
+import AuthenticationServices
+
+/// Singleton helper to manage
```

Comment:
I don't think this is a singleton anymore

Reply:
Good catch – I didn't even finish that sentence 🤦‍♀️ 

Updated